ansible/roles/kubernetes-addons/templates/dns/kube-svc.yaml.j2

```diff
@@ -1,4 +1,4 @@
-# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/skydns-svc.yaml
+# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/kubedns-svc.yaml
```

Comment:
This file was renamed to `kube-svc.yaml.j2`  shouldn't it be `kubeDNS-svc.yaml.j2` ?